config/application.rb

```diff
@@ -1,6 +1,7 @@
 require File.expand_path('../boot', __FILE__)
 
 require 'rails/all'
+require 'coffee_script'
```

Comment:
I thought so, but it goes beyond the scope of this PR. We can create an issue to fix this